app/lib/payment.rb

```diff
@@ -0,0 +1,16 @@
+class Payment
+
+  def payment_result(money_tendered, receipt_total)
+    change = calculate_change(money_tendered, receipt_total)
+    if  change >= 0
+      "Change - #{change}"
```

Comment:
I wouldn't return strings, normally we want to avoid this and instead return the information as a hash that can be handled by our presentation layer